### PR TITLE
Fixed sign message text [#4529]

### LINKED ISSUE #4529: There is no message on sign message screen

### PR DESCRIPTION
**Pull request by flexsurfer**
fixes #4529

### CODE REVIEW

File: src/status_im/ui/screens/wallet/send/views.cljs [hunk 1]
```diff
@@ -282,9 +282,11 @@
        [react/view styles/send-transaction-form
         [wallet.components/cartouche {:disabled? true}
          (i18n/label :t/message)
-         [components/amount-input {:disabled?     true
-                                   :input-options {:multiline     true
-                                                   :default-value data}} nil]]]]
+         [components/amount-input
+          {:disabled?     true
+           :input-options {:multiline true}
+           :amount-text   data}
```
Comment (flexsurfer) [inline src/status_im/ui/screens/wallet/send/views.cljs:288]: this is a fix, it was a bad idea to use amount-input for a message

**Review by jeluard — APPROVED** [review]

File: src/status_im/ui/screens/wallet/send/views.cljs (hunk 1 above)
Comment (goranjovic) [inline src/status_im/ui/screens/wallet/send/views.cljs:288]: This is ok for a hotfix, but this message deserves its own component.

**Review by goranjovic — APPROVED** [review]

File: src/status_im/ui/screens/wallet/send/views.cljs (hunk 1 above)
Comment (goranjovic) [reply src/status_im/ui/screens/wallet/send/views.cljs:288]: My comment - same thing :)

Comment (flexsurfer) [reply src/status_im/ui/screens/wallet/send/views.cljs:288]: Fully agree

**Comment by rasom** [comment]
apk uploaded to https://i.diawi.com/y8Jj18 for branch PR-4531

<blockquote><div><strong><a href="https://i.diawi.com/y8Jj18">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

**Comment by rasom** [comment]
ipa uploaded to https://i.diawi.com/81kAA4 for branch PR-4531

<blockquote><div><strong><a href="https://i.diawi.com/81kAA4">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

**Comment by annadanchenko** [comment]
@yevh-berdnyk please confirm that failing 'test_open_transaction_on_etherscan' is not caused by this PR, so we can merge or explain what's introduced if PR is causing it.

I've checked initial issue and sending ETH/tokens from wallet, DApp and all seem fine, so waiting for feedback on failing autotest

**Comment by yevh-berdnyk** [comment]
@annadanchenko yes, this issue is not caused by the PR, it's an ainfrastructure issue - device was not started by some reason

**Pull request merged**